src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -138,7 +138,16 @@ network_id | Network/@id | This field is prefixed. The technical part of the NeT
 network_name | Network/Name | 
 network_timezone | | Fixed value `Europe/Paris`.
 
-The `network` has a complementary code `source` with the identifier of the original associated `Network`.
+In the case of a line without an associated network, the associated operator is
+used as a network. If there is no associated operator, the line is ignored.
+
+NTFS field | Netex-IDFM element | Mapping rule/Comment
+--- | --- | ---
+network_id | Operator/@id | This field is prefixed. 
```

Comment:
With ```Operator_``` concatenation

Reply:
Done in 8fbb32d94beaddb42abad4c24938e430ffdf33c6.